src/hooks/polkadotJs/tests/mockUsePolkadotJsContext.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiPromise } from '@polkadot/api';
+import { AccountId32 } from '@polkadot/types/interfaces';
+
+export const nativeAssetBalance = '10';
+export const nonNativeAssetBalance = '20';
+
+export const getMockApiPromise = () =>
+  ({
+    query: {
+      system: {
+        account: jest.fn((arg: AccountId32 | string | Uint8Array) => {
```

Comment:
args are not used here, please remove or prefix with `_args`. Also unsure why does this pass the linter checks.